packages/flutter/lib/src/services/scribe.dart

```diff
@@ -0,0 +1,129 @@
+  ///    Flutter's built-in text fields support handwriting input.
+  static Future<bool?> isFeatureAvailable() {
+    return _channel.invokeMethod<bool?>(
+      'Scribe.isFeatureAvailable',
+    );
+  }
+
+  /// Returns true if the InputMethodManager supports Scribe stylus handwriting
+  /// input, false otherwise.
+  ///
+  /// Call this each time before calling [startStylusHandwriting] to make sure
+  /// it's available.
+  ///
+  /// The return type is nullable due to the way
+  /// [MethodChannel.invokeMethod](https://main-api.flutter.dev/flutter/services/MethodChannel/invokeMethod.html)
+  /// works, but a successful call will never resolve to `null`.
+  ///
+  /// Supported on Android API 34 and above. If called by an unsupported API
+  /// level, a [PlatformException] will be thrown. To avoid error handling, use
+  /// the convenience method [isFeatureAvailable] instead.
+  ///
+  /// {@tool snippet}
+  /// This example shows using [isStylusHandwritingAvailable] to confirm that
+  /// [startStylusHandwriting] can be called.
+  ///
+  /// ```dart
+  /// try {
+  ///   if (!(await Scribe.isStylusHandwritingAvailable() ?? false)) {
+  ///     // If isStylusHandwritingAvailable returns false then the device's API level
+  ///     // supports Scribe, but for some other reason it's not able to accept stylus
+  ///     // input right now.
+  ///     return;
+  ///   }
+  /// } on PlatformException catch (exception) {
+  ///   if (exception.message == 'Requires API level 34 or higher.') {
+  ///     // The device's API level is too low to support Scribe.
+  ///     return;
+  ///   }
+  ///   // Any other exception is unexpected and should not be caught here.
+  ///   rethrow;
+  /// }
+  ///
+  /// // Scribe is supported, so start it.
+  /// Scribe.startStylusHandwriting();
+  /// ```
+  /// {@end-tool}
+  ///
+  /// See also:
+  ///
+  /// * <https://developer.android.com/reference/android/view/inputmethod/InputMethodManager#isStylusHandwritingAvailable()>,
+  ///   which is the corresponding API on Android.
+  static Future<bool?> isStylusHandwritingAvailable() {
+    return _channel.invokeMethod<bool?>(
+      'Scribe.isStylusHandwritingAvailable',
+    );
+  }
+
+  /// Tell Android to begin receiving stylus handwriting input.
+  ///
+  /// This is typically called after detecting a [PointerDownEvent] on an active
+  /// text field, indicating the start of stylus handwriting input. If there is
+  /// no active [TextInputConnection], the call will be ignored.
+  ///
+  /// Supported on Android API 33 and above.
+  ///
+  /// See also:
+  ///
+  /// * <https://developer.android.com/reference/android/view/inputmethod/InputMethodManager#startStylusHandwriting(android.view.View)>,
+  ///   which is the corresponding API on Android.
+  ///  * [EditableText.stylusHandwritingEnabled], which controls whether
+  ///    Flutter's built-in text fields support handwriting input.
+  static Future<void> startStylusHandwriting() {
```

Comment:
so the IME is responsible for determining when to stop?

Reply:
Yes. It seems like it's a timeout from the last screen touch.